### DISCUSSION TITLE
Angular: TanStackField type

### DISCUSSION THREAD

**Discussion by GonzaloManoukian** [TanStack/form#888] [Q&A]
I have a custom input and i need to pass the tanstackField by prop like this:

```
      <ng-container
        [tanstackField]="loginForm"
        name="user"
        #user="field"
        [validators]="{ onBlur: userSchema.shape.user }"
      >
        <app-input-password
          [formField]="user"
          label="Email"
          [required]="true"
        />
      </ng-container>
```

In my InputPasswordComponent im trying to do something like this:

```
import { TanStackField } from '@tanstack/angular-form';

type TanstackType = typeof TanStackField;

export class InputPasswordComponent {
  formField = input<TanstackType>();
  }
```

Ive got type errors, i tried a lot of stuffs and nothing works (only any). Is there something usefull for this case?

Thanks!

**Comment by Balastrong** [comment]
Can you please share a minimal reproducible example by forking one of the form examples in the docs? (e.g. https://tanstack.com/form/latest/docs/framework/angular/examples/simple)

**Comment by Aqib-Rime** [comment]
any update on this? @Balastrong

**Reply by crutchcorn** [reply]
Please do not tag specific maintainers
